### DISCUSSION TITLE
游戏启动前执行命令

### DISCUSSION THREAD

**Discussion by ZS2147483647** [Meloong-Git/PCL#3677] [提问] [answered]
设置游戏启动器执行命令dir "{verindie}mods" /b | clip(目的是获取mod列表剪贴板)hmcl启动器报错,pcl可以正常运行
如何进行等价转化

**Comment by zkitefly** [comment]
在游戏目录下写个 bat 脚本，应该 HMCL 和 PCL 都能操作？

**Comment by wuliaodexiaoluo** [comment]
请参考 HMCL 启动器的提示，PCL 的命令不一定适用于 HMCL
建议直接 bat 得了

**Reply by wuliaodexiaoluo** [reply]
cmd -c [文件名].bat 参数
dir %1

**Comment by burningtnt** [comment, marked as answer]
加上 /C 即可。写：`cmd /C "dir mods | clip"`

**Comment by allMagicNB** [comment]


**Reply by wuliaodexiaoluo** [reply]
& 不能替代管道符吧。。。
